k8s/tools/admin-node/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:18.04
+
+RUN apt-get update\
+        && apt-get install -y --no-install-recommends mysql-client python-setuptools python-pip\
+        && pip install awscli \
```

Comment:
AWS [recommends](https://docs.aws.amazon.com/cli/latest/userguide/installing.html) adding `--upgrade` to the install, so:
```
& pip install --upgrade awscli \
```